src/index.ts

```diff
@@ -199,6 +276,14 @@ export class WaxJS {
   private receiveLogin(data: ILoginResponse): void {
     this.user = data;
 
+    if (Object.keys(data.sideChainAccount || {}).includes(this.chainName)) {
+      const sideChain = data.sideChainAccount[this.chainName];
```

Comment:
should `sideChain` be renamed `sideChainAccount`? The way it is expressed makes me think this is an actual sidechain. Even just naming it `account` makes more sense to me

Reply:
updated 👍